plone/app/upgrade/v51/configure.zcml

```diff
@@ -233,4 +233,16 @@ Add image scaling options to image handling control panel.
 
     </gs:upgradeSteps>
 
+    <gs:upgradeSteps
+        source="5111"
+        destination="5112"
+        profile="Products.CMFPlone:plone">
+
+        <gs:upgradeStep
+            title="Update an action i18n domain"
+            handler=".final.fix_i18n_domain"
+            />
+
+    </gs:upgradeSteps>
```

Comment:
The `upgradeStep` (singular) is good, but with this we get two tags `gs:upgradeSteps` (plural) with the same source and destination. They should be merged into one `gs:upgradeSteps` tag with one `upgradeDepends` and one `upgradeStep`.

I would expect an error when Plone starts up with this, but maybe the GenericSetup zcml is forgiving.

For the rest: LGTM.